go/host/p2p/p2p.go

```diff
@@ -356,12 +355,15 @@ func (p *Service) broadcast(msg message) error {
 	copy(currentAddresses, p.peerAddresses)
 	p.peerAddressesMutex.RUnlock()
 
-	var wg sync.WaitGroup
 	for _, address := range currentAddresses {
-		wg.Add(1)
-		go p.sendBytesWithRetry(&wg, address, msgEncoded) //nolint: errcheck
+		closureAddr := address
```

Comment:
what was the purpose of the WaitGroup?

Reply:
I believe it was to issue the payload to all nodes in parallel and wait until the whole broadcast was finished.
With a 2 min retry and one node with p2p off, this had a weird effect on the sequencer which would not update the host db in proper time. 

When running the sim with this code, all nodes (host db) have height ~80 but the sequencer (host db) stays at high ~60